src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -187,7 +229,16 @@ class Simplex_tree {
   /** \brief Range over the vertices of a simplex. */
   typedef boost::iterator_range<Simplex_vertex_iterator> Simplex_vertex_range;
   /** \brief Range over the cofaces of a simplex. */
-  typedef std::vector<Simplex_handle> Cofaces_simplex_range;
+  typedef typename std::conditional<Options::link_nodes_by_label,
+                                    Optimized_star_simplex_filtered_range,  // faster implem
+                                    std::vector<Simplex_handle>>::type Cofaces_simplex_range;
+
+  // 40 seems a conservative bound on the dimension of a Simplex_tree for now, as it would not fit on the biggest
+  // hard-drive.
+  // static_vector still has some overhead compared to a trivial hand-made
+  // version using std::aligned_storage, or compared to making suffix_ static.
+  using Static_vertex_vector = boost::container::static_vector<Vertex_handle, 40>;
```

Comment:
`/// @private` or something to hide it in the doc?

Reply:
Marked as private on [0bb0bd3](https://github.com/GUDHI/gudhi-devel/pull/877/commits/0bb0bd3bdab957c1cf77a2d0649f7397c8ab22b9)